client/src/components/modals_and_popovers/FocusLockedModal.stories.js

```diff
@@ -0,0 +1,53 @@
+import { useArgs } from "@storybook/client-api";
+import React, { Fragment } from "react";
+
+import { FocusLockedModal } from "./FocusLockedModal";
+
+export default {
+  title: "modals and popovers/FocusLockedModal",
+  component: FocusLockedModal,
+
+  // Need decorators to use useArgs()
+  decorators: [(Story) => <div>{Story()}</div>],
+};
+
+const Template = (args) => {
+  const [_, updateArgs] = useArgs();
+  function on_exit() {
+    console.log(args.mounted ? "Closing Modal" : "Opening Modal");
+    updateArgs({ ...args, mounted: !args.mounted });
+  }
+
+  return (
+    <Fragment>
+      <div id="ib-site-header-area" />
+      <div style={{ height: "500vh" }}> Switch mounted control to focus </div>
+      {args.mounted ? null : (
+        <button onClick={on_exit}>Click to open modal</button>
+      )}
```

Comment:
Same as comment in `FixedPopover`

Reply:
Additionally, to demonstrate the focus lock, would have some focusable items both inside and outside the modal

Reply:
Done!